components/02-molecules/meta/basic-meta/_yds-basic-meta.scss

```diff
@@ -20,4 +20,10 @@
       border-left: var(--border-thickness-1) solid;
     }
   }
+
+  .date-time {
+    font-variant: small-caps;
+    font-variant-numeric: oldstyle-nums;
+    text-transform: lowercase;
+  }
```

Comment:
This can be removed, now that you added these styles to the `.date-time` element in the other file. If you inspect your browser, you'll see that this is overriding those styles, with exactly the same styles. So it's not providing any benefit here.